### PR TITLE
[BidManager] cleanup

### PR DESCRIPTION
**Pull request by vguerci**

### CODE REVIEW

**Comment by vguerci** [comment]
Less code, more tests 🐱

File: CriteoPublisherSdk/Sources/Criteo.m [hunk 1]
```diff
@@ -105,28 +101,19 @@ - (void)registerCriteoPublisherId:(NSString *)criteoPublisherId
 - (void)_registerCriteoPublisherId:(NSString *)criteoPublisherId
                        withAdUnits:(NSArray<CRAdUnit *> *)adUnits {
   self.config.criteoPublisherId = criteoPublisherId;
-
-  CR_CacheAdUnitArray *cacheAdUnits = [CR_AdUnitHelper cacheAdUnitsForAdUnits:adUnits];
-  [self.registeredAdUnits addObjectsFromArray:cacheAdUnits];
   [self.appEvents registerForIosEvents];
   [self.appEvents sendLaunchEvent];
   [self.configManager refreshConfig:self.config];
   if (!self.config.liveBiddingEnabled) {
```
Comment (TurpIF) [inline CriteoPublisherSdk/Sources/Criteo.m:107]: Not part of this PR, but shouldn't we remove the prefetch only after an A/B test ?

**Review by TurpIF — APPROVED** [review]
LGTM, even if I will miss the TTNC

File: CriteoPublisherSdk/Tests/UnitTests/CR_BidManagerTests.m [hunk 2]
```diff
@@ -181,7 +178,7 @@ - (void)testRegistrationSetEmptyBid {
   XCTAssert(bid.isEmpty);
 }
 
-- (void)testGetBidWhenBeforeTtnc {  // TTNC -> Time to next call
+- (void)testGetBidWhenBeforeTimeToNextCall {
```
[the remark this answers is not in the document]
Comment (vguerci) [reply CriteoPublisherSdk/Tests/UnitTests/CR_BidManagerTests.m:173]: I don't like that comment that point to a discutable naming choice ;)

File: CriteoPublisherSdk/Sources/Criteo.m (hunk 1 above)
Comment (vguerci) [reply CriteoPublisherSdk/Sources/Criteo.m:107]: This does not remove the prefetch, just disable it when live bidding is enabled? Which is in fact part of the A/B testing, isn't?

**Comment by vguerci** [comment]
Thanks for your review! 🙏

File: CriteoPublisherSdk/Sources/Criteo.m (hunk 1 above)
Comment (vguerci) [reply CriteoPublisherSdk/Sources/Criteo.m:107]: As discussed, will submit another PR to remove that check.
